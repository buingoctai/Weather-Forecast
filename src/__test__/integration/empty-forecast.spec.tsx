import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import EmptyForecast from 'srcRoot/pages/WetherForecast/components/list-forecast/components/empty-forecast';

let emptyElement: RenderResult;
describe('Testing <EmptyForecast/>', () => {
  beforeEach(() => {
    emptyElement = render(<EmptyForecast />);
  });

  it('Matches snapshot', () => {
    const { baseElement } = emptyElement;
    expect(baseElement).toMatchSnapshot();
  });
});
