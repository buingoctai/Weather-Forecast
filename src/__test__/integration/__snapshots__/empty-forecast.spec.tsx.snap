// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <EmptyForecast/> Matches snapshot 1`] = `
<body>
  <div>
    <div
      class="flx-center list-forecast flx-col"
    >
      <img
        height="80"
        src="test-file-stub"
        width="80"
      />
      <h4
        style="font-size: 15px;"
      >
        Empty Weather Forecast
      </h4>
    </div>
  </div>
</body>
`;
